express/scripts/instrument.js

```diff
@@ -53,6 +55,24 @@ if (window.spark && window.spark.hostname === 'www.stage.adobe.com') {
   martechURL = 'https://www.adobe.com/marketingtech/main.stage.min.js';
 }
 
+// alloy feature flag
+if (
+  (
+    martech === 'alloy'
+    && window.spark
+    && window.spark.hostname === 'www.stage.adobe.com'
+  )
+  || (
+    martech === 'alloy-qa'
+  )
+) {
+  martechURL = 'https://www.adobe.com/marketingtech/main.standard.qa.js';
+} else if (
+  martech === 'alloy'
+) {
+  martechURL = 'https://www.adobe.com/marketingtech/main.standard.min.js';
+}
+
```

Comment:
Please follow the `airbnb` coding style.

Reply:
Never mind, according to https://github.com/airbnb/javascript#control-statements this pattern is actually allowed. Thanks @bbythewa 😄